src/components/RichTextEditor.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { EditorContent, useEditor } from '@tiptap/react';
+import DOMPurify from 'dompurify';
+
+import StarterKit from '@tiptap/starter-kit';
+import TaskItem from '@tiptap/extension-task-item';
+import TaskList from '@tiptap/extension-task-list';
+import Highlight from '@tiptap/extension-highlight';
+import TextAlign from '@tiptap/extension-text-align';
+
+// eslint-disable-next-line import/extensions
```

Comment:
why did you disabled this rule?

Reply:
check the other rules that you have disabled, try not to disable them, if you have to disable them tell me why